src/app/components/common/delete/delete.component.html

```diff
@@ -0,0 +1,8 @@
+<h2 class="govuk-heading-m">Are you sure you want to delete {{ numberOfItems > 1 ? 'these items' : 'this item' }}?</h2>
+
+<ng-content></ng-content>
```

Comment:
Do you need this bit?

Reply:
<ng-content> ? yes

Reply:
<app-delete> Whatever we pass in here gets spat out out where ng-content is </app-delete>